devicon.json

```diff
@@ -11278,12 +11278,26 @@
                 "plain-wordmark"
             ]
         },
-        "color": "#f7a41d",
-        "aliases": [
-            {
-                "base": "original",
-                "alias": "plain"
-            }
-        ]
+        "color": "#294172", 
+        "aliases": [] 
+    },
+    {
+        "name": "stackblitz",
+        "tags": [
+            "IDE",
+            "FullStack Development Online",
+            "online ide"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#587dbd",
+        "aliases": ["stack-blitz"]
```

Comment:
This is not what the aliases object is used for. Please read see the wiki for more info: on updating `devicon.json`: https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60